feat(non_lts_galactic): add example (#8)
ubuntu-robotics/ros-snaps-examples: non_lts_galactic/README.md

# Non LTS ROS snap

> via [extensions](https://snapcraft.io/docs/supported-extensions), snapcraft offers support LTS versions of ROS.

Reviewer: ```suggestion
snapcraft offers support for LTS versions of ROS via [extensions](https://snapcraft.io/docs/supported-extensions). They factorize some YAML code that are common to ROS snaps. While convenient, they are not necessary.
```

Reviewer: Note that the updated link does not exists yet.

Author: Shouldn't we use the original link for the time being?

Reviewer: ~~I've edited the initial suggestion.~~

> snapcraft offers support for LTS versions of ROS via [extensions](https://snapcraft.io/docs/supported-extensions), They factorize some YAML code that are common to ROS snaps. While convenient, they are not necessary.

Reviewer: ```suggestion
Snapcraft offers support for LTS versions of ROS via [extensions](https://snapcraft.io/docs/supported-extensions).
They factorize some YAML code that are common to ROS snaps. While convenient, they are not mandatory.
```

> With some extra work you can still build a ROS snap without an extensions. Here we give an example by building a ROS 2 Galactic snap.

Reviewer: ```suggestion
This example demonstrates how to build a ROS 2 snap based on Galactic, a non-LTS distribution, thus without using an extension.
```
